derive_client/clients/base_client.py

```diff
@@ -19,10 +19,10 @@
     WithdrawModuleData,
 )
 from derive_action_signing.signed_action import SignedAction
-from derive_action_signing.utils import MAX_INT_32, get_action_nonce, sign_rest_auth_header, sign_ws_login, utc_now_ms
+from derive_action_signing.utils import MAX_INT_32, get_action_nonce, sign_rest_auth_header, utc_now_ms
 from pydantic import validate_call
 from web3 import Web3
-from websocket import WebSocketConnectionClosedException, create_connection
+from websocket import create_connection
```

Comment:
hmm still need to (re)move this

https://github.com/8ball030/derive_client/blob/e74ccf937081c444d0466b088303b9b1c13960a4/derive_client/clients/base_client.py#L123-L125